ci/helm/values/env-values-nuxeo.yaml.gotmpl

```diff
@@ -0,0 +1,22 @@
+nuxeo:
+  image:
+    repository: {{ printf "%s/nuxeo/nuxeo-coldstorage-ftests" (requiredEnv "DOCKER_REGISTRY") }}
+    tag: {{ requiredEnv "VERSION" | quote }}
+  amazonS3:
+    enabled: true
+    auth:
+      accessKeyId: {{ requiredEnv "AWS_ACCESS_KEY_ID" }}
+      secretKey: {{ requiredEnv "AWS_SECRET_ACCESS_KEY" }}
+    region: {{ requiredEnv "AWS_REGION" }}
+    bucket: {{ requiredEnv "AWS_BUCKET" }}
+    bucketPrefix: {{ requiredEnv "AWS_BUCKET_PREFIX" }}
+  customEnvs:
+  {{- if (contains "preprod" (env "CONNECT_CLID_SECRET")) }}
+  - name: NUXEO_CONNECT_URL
+    value: https://nos-preprod-connect.nuxeocloud.com/nuxeo/site/
+  {{- end }}
+  - name: NUXEO_CLID
+    valueFrom:
+      secretKeyRef:
+        name: {{ env "CONNECT_CLID_SECRET" | default "instance-clid" }}
+        key: instance.clid
```

Comment:
Probably not needed anymore, sonce we are now installing packages at build time.